info/v1/container.go

```diff
@@ -72,6 +72,10 @@ type ContainerReference struct {
 	// Namespace under which the aliases of a container are unique.
 	// An example of a namespace is "docker" for Docker containers.
 	Namespace string `json:"namespace,omitempty"`
+
+	Image string `json:"image,omitempty"`
```

Comment:
These should not be in the reference but rather in the ContainerSpec above (and Metadata should be merged with labels I would think)